hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupSystemTable.java

```diff
@@ -1720,7 +1726,8 @@ public String[] getListOfBackupIdsFromMergeOperation() throws IOException {
       if (val.length == 0) {
         return null;
       }
-      return new String(val).split(",");
+      List<String> result = Splitter.on(',').splitToList(new String(val, StandardCharsets.UTF_8));
```

Comment:
Splitter.on(',').splitToStream(new String(val, StandardCharsets.UTF_8)).toArray(String[]::new)

Reply:
Ok, seems fine to do it with streams